pkg/controller/serviceexternalip/controller_test.go

```diff
@@ -303,6 +453,7 @@ func TestSyncService(t *testing.T) {
 }
 
 func checkForServiceExternalIP(t *testing.T, controller *loadBalancerController, name, namespace, expectedExternalIP string) {
+	t.Helper()
```

Comment:
nice, I always forget about that functionality

Reply:
I keep forgetting it too. If I add it in a PR, I must have tried find out which caller actually failed.